tests/Examples/V3/ExampleTestsUtils.cs

```diff
@@ -167,23 +208,39 @@ internal static string CreateAdGroup(GoogleAdsClient client, string campaignReso
         /// <returns>The resource name of the newly created expanded text ad.</returns>
         internal static string CreateExpandedTextAd(GoogleAdsClient client,
             string adGroupResourceName)
+        {
+            Ad ad = new Ad
+            {
+                FinalUrls = { "http://www.example.com/" },
+                ExpandedTextAd = new ExpandedTextAdInfo
+                {
+                    Description = "Buy your tickets now!",
+                    HeadlinePart1 = $"Cruise #{DateTime.Now.Ticks % 1000} to Mars",
+                    HeadlinePart2 = "Best Space Cruise Line",
+                    Path1 = "path1",
+                    Path2 = "path2"
+                }
+            };
+
+            return CreateAdGroupAd(client, ad, adGroupResourceName);
+        }
+
+        /// <summary>
+        /// Creates an ad group ad.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="adGroupResourceName">Resource name of the ad group in which expanded
```

Comment:
The comment seems wrong to me: this can be used for any ad, not only an expanded text ad

Reply:
Maybe just change it to 'in which the ad is created'.

Reply:
Done.